spec/support/request_helpers.rb

```diff
@@ -89,23 +89,11 @@ def check_400_on_invalid_json
     end
   end
 
-  def check_content_type_header
```

Comment:
Summary of in-person discussion with Elliot just now:

Looks as though the golang publishing api used to propagate the body of the response from content-store and that's why it was modified to also propagate the headers. The ruby version does not propagate the response body from content-store so it does not make sense to propagate the content type header.

There may still be some error handling edge cases not properly handled, I'm going to review that.